IBNet/TWSEncoding.cs

```diff
@@ -89,11 +173,29 @@ internal EnumEncDec(Type t)
     }
 
     const string IB_EXPIRY_DATE_FORMAT = "yyyyMMdd";
-    readonly string NUMBER_DECIMAL_SEPARATOR;
 
     static readonly Dictionary<Type, EnumEncDec> _enumDecoders;
 
-    protected Stream _stream;
+    protected Stream Stream;
+
+    int _inEncodingBatch;
+    protected bool InEncodingBatch
```

Comment:
I really don't like the `Batch` naming.
It's meaningless in this context.
Why not just use something close to what we are doing here like `Message`?